observation_portal/proposals/serializers.py

```diff
@@ -53,7 +53,7 @@ class ProposalInviteResponseSerializer(serializers.Serializer):
 class MembershipSerializer(serializers.ModelSerializer):
     class Meta:
         model = Membership
-        fields = ('id', 'proposal', 'role', 'user', 'time_limit')
+        fields = ('id', 'proposal', 'role', 'user', 'time_limit','active')
```

Comment:
active is a field on the proposal, not on the membership... Why are you adding it here?

Reply:
I have no idea where this comes from. I have no reason to add this.